Messenger/AppDelegate.mm

```diff
@@ -120,7 +121,7 @@ - (void)applicationDidFinishLaunching:(NSNotification*)notification {
   if ([[NSUserDefaults standardUserDefaults] boolForKey:@"moves-with-active-space"]) {
     _window.collectionBehavior |= NSWindowCollectionBehaviorMoveToActiveSpace;
   }
-  _window.minSize = {605,300};
+  _window.minSize = {400,300};
```

Comment:
I've been using this build for a few weeks now without seeing any behaviour like that, and this is on OS X 10.11. What version did you notice the scrolling on?